osctiny/extensions/packages.py

```diff
@@ -237,13 +237,20 @@ def push_file(self, project, package, filename, data):
         :param filename: Name of file
         :param data: content of file
         :type data: str or open file handle
+        :param comment: Optional comment to use as commit message
+
+        .. versionchanged:: {{ NEXT_RELEASE }}
+
+           Added an optional `comment` argument to be used as the commit message when writing the
```

Comment:
Sorry, this is RST :wink: 

```suggestion
           Added an optional ``comment`` argument to be used as the commit message when writing the
```